docs/workshops/intro-to-accessibility.md

```diff
@@ -72,8 +69,392 @@ Try using your phone with the screen reader function.
 
 Read about using a switch
 
-https://www.24a11y.com/2018/i-used-a-switch-control-for-a-day/
+[www.24a11y.com/2018/i-used-a-switch-control-for-a-day/](https://www.24a11y.com/2018/i-used-a-switch-control-for-a-day/)
 
 Watch some of the videos on this page to understand more about how people with disabilities use tech.
 
-https://axesslab.com/tech-youtubers/
+[axesslab.com/tech-youtubers/](https://axesslab.com/tech-youtubers/)
+
+---
+
+## Workshop
+
+### Contents
+
+- Different aspects of accessibility
+  - Vision impairments
+  - Hearing impairments
+  - Motor impairments
+  - Assistive devices
+  - Cognitive impairments
+  - Invisible disabilities
+- Why it's important to consider
+- How to make your social media posts accessible
+- How to make your documents accessible
+- WCAG (Web Content Accessibility Guidelines)
+
+---
+
+### Different aspects of accessibility
+
+What is digital accessibility?
+
+"Digital accessibility is the ability of a website, mobile application or electronic document to be easily navigated and understood by a wide range of users, including those users who have visual, auditory, motor or cognitive impairments."
+
+**In the UK, 1 in 5 people are disabled**
+
+The concept of accessibility doesn’t just apply to disabled people – all users will have different needs at different times and in different circumstances.
+
+**Making things more accessible makes things better for all users.**
+
+_"My needs are different from yours and my needs today might be different tomorrow”_ - Charles Hall
+
+#### Vision impairments
+
+In the UK: almost 2 million people living with sight loss.
+360,000 are registered as blind or partially sighted.
+
+**Blindness**: there is a legal definition which doesn’t imply seeing nothing at all. Many can read from a screen, but can’t use a mouse.
+
+They often can’t see images, graphics or videos.
+
+If the layout of your page, the navigation or labels of your fields are not clear, they won’t be able to use your website or app.
+
+The main screen readers are:
+
+- NVDA - it’s free
+- JAWS - need a licence but good and have been here for a long time
+- Voice Over
+
+2 minutes video of a user (Kristy Viers) using an iPhone
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/l0xBsR5mIYs" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+#### People with low vision
+
+- Might need to magnify and miss some information if you don’t **align your content on the left** or don’t group related items together.
```

Comment:
Is this dependent on the left-to-right or right-to-left setting on the browser?

Reply:
yes, I'm assuming left to right languages (which would mean probably the user would have that setting in their browser) maybe we could add:  (for left to right language - otherwise you would have to align on the right) ?